### ISSUE TITLE
Request: Show C++ default member values when INLINE_SIMPLE_STRUCTS=YES

### ISSUE THREAD

**Issue by lamm45** [doxygen/doxygen#9394]
If a struct in C++ has default member initializers, their values are often the most helpful information to the user. However, it seems that the default values are currently not included in the doxygen output if INLINE_SIMPLE_STRUCTS is set to YES.

Consider the following example:
```cxx
namespace mylib {

struct Settings {
    int min_brightness = 20; ///< Minimum brightness
    int max_brightness = 127; ///< Maximum brightness
};

} // namespace mylib
```

When INLINE_SIMPLE_STRUCTS=NO, the output with doxygen 1.8.17 looks like this:
![INLINE_SIMPLE_STRUCTS=NO](https://user-images.githubusercontent.com/96844552/173236403-5ac86c47-1ca2-4ee8-85f5-30b80c0c2269.png)

When INLINE_SIMPLE_STRUCTS=YES, the output with doxygen 1.8.17 looks like this:
![INLINE_SIMPLE_STRUCTS=YES](https://user-images.githubusercontent.com/96844552/173236404-88f49fdb-feb6-4e99-9bcd-de98a78dd867.png)

I think it would make sense to show the default values in the inlined version as well. Maybe after an equality sign (as in the non-inlined version), or perhaps in a new separate column.

**Comment by doxygen** [comment]
@lamm45 I've implemented the suggestion in the referenced commit. Please check if it looks as expected.

**Comment by lamm45** [comment]
Looks perfect. Thanks!

![new_INLINE_SIMPLE_STRUCTS=YES](https://user-images.githubusercontent.com/96844552/177887829-ee48bdd8-f9ef-40a2-bf19-aeea19beb768.png)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.5.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
